SPECS/libcontainers-common/libcontainers-common.spec

```diff
@@ -16,35 +16,35 @@
 #
 
 # commonver - version from containers/common
-%define commonver 0.44.0
+%define commonver 0.57.4
+# imagever - version from containers/image
+%define imagever 5.29.2
 # podman - version from containers/podman
-%define podmanver 3.3.1
+%define podmanver 4.9.3
 # storagever - version from containers/storage
-%define storagever 1.36.0
-# imagever - version from containers/image
-%define imagever 5.16.0
+%define storagever 1.52.0
 Summary:        Configuration files common to github.com/containers
 Name:           libcontainers-common
-Version:        20210626
-Release:        2%{?dist}
+Version:        20240213
+Release:        1%{?dist}
 License:        ASL 2.0 AND GPLv3
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          System/Management
 URL:            https://github.com/containers
-#Source0:       https://github.com/containers/image/archive/refs/tags/v5.16.0.tar.gz
```

Comment:
While changing these, I'd prefer to moving to the model where we rename the file in the `Source` line, like this:
```
Source0:       https://github.com/containers/image/archive/refs/tags/v5.16.0.tar.gz#/%{name}-image-%{imagever}.tar.gz
```

Naturally, that goes for all of the sources that are done this way.